docs/tutorial/tutorial-3.rst

```diff
@@ -386,59 +386,115 @@ or doing other pre-distribution tasks.
   .. group-tab:: Linux
 
     The output of the package step will be slightly different depending on
-    your Linux distribution. If you're on a Debian-derived distribution,
-    you'll see:
+    your Linux distribution. You'll see the following output, if you're on a system that is:
 
-    .. code-block:: console
+    .. tabs::
 
-      (beeware-venv) $ briefcase package
+      .. group-tab:: Debian Based
```

Comment:
Tabs inside tabs isn't a great reading experience. The choice to have 3 (relatively terse) blocks rather than nested tabs was deliberate.